### DISCUSSION TITLE
non explicit reference to _quarto.yaml

### DISCUSSION THREAD

**Discussion by ShKlinkenberg** [quarto-dev/quarto-cli#3144] [Q&A] [answered]
I make a lot of use of the _quarto.yaml file to set up general settings. This way, all documents, for example, presentations share the same settings, and I do not have to add the yaml to each file. Though this is very convenient, I do not get why this dependency is implicit. I would rather explicitly refer to the relevant yaml file in the yaml header of a document. Is this possible?

That way, I could even share yaml templates with others in my organization. For example through a public URL.

**Comment by dragonstyle** [comment, marked as answer]
We don't currently support sharing settings via things like URL directly reference in YAML, but you can include external YAML files via:

https://quarto.org/docs/projects/quarto-projects.html#metadata-includes

In addition, I think it would be worth considering whether it would make sense to create a new format for your document types. This has more overhead than a simple metadata file but will make usage explicit and allow the document to conditionally target that output format. 

https://quarto.org/docs/extensions/formats.html

HTH!

**Comment by mcanouil** [comment]
If I may, `_quarto.yml` defines a Quarto project and settings that should/need to be shared within this project.
It seems you are or want to use `_quarto.yml` as some kind of user setting file.

Maybe, the solution or a solution would be to have different layer of configuration:
1. Global level (settings stored somewhere "neutral")[^1]
2. User level (settings stored in home directory)[^1]
3. Project level (this is the current use of `_quarto.yml`)
4. File level (this already exists, since metadata can be set in the yaml header)

Note on sharing through URL, for reproducibility and security reasons, the settings should be from a local file, even if user has to copy/paste the content or download the file from an URL.

[^1]: These user/global level settings also break the "reproducibility" part since users might easily forget to share those settings with the files.
